common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4070,6 +4070,12 @@
     ],
     "sqlState" : "HV091"
   },
+  "NON_DETERMINISTIC_CHECK_CONSTRAINT" : {
+    "message" : [
+      "The check constraint `<checkCondition>` is non-deterministic. Check constraints must only contain deterministic expressions."
+    ],
+    "sqlState" : "42621"
```

Comment:
The error code seems consistent with DB2 and what we use for generated columns, +1.